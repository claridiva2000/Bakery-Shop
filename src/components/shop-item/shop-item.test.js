import React from 'react';
import renderer from 'react-test-renderer';
import ShopItem from './shop-item.component';

it('should render DirectoryItem component', () => {
  const component = renderer.create(<ShopItem />).toJSON();
  expect(component).toMatchSnapshot();
});
