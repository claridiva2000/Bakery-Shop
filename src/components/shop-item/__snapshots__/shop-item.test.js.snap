// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DirectoryItem component 1`] = `
<div
  className="shopItem"
>
  <div
    className="image"
  >
    <img
      alt="undefined"
      src="undefined"
    />
  </div>
  <h2 />
  <div
    className="price"
  >
    <span>
      $undefined full
    </span>
    <span>
      $undefined slice
    </span>
  </div>
  <button
    className="custom-button "
  >
    Add to Cart
  </button>
</div>
`;
